contrib/podmanimage/stable/Containerfile

```diff
@@ -0,0 +1,55 @@
+#
```

Comment:
Right, since they're mostly the same.  Oof, I'm left really wanting something like an `#include` directive to help cut down on the duplication :confounded: